examples/progress.js

```diff
@@ -0,0 +1,43 @@
+var http = require('http')
+var multiparty = require('../')
+var util = require('util')
+
+var PORT = process.env.PORT || 8080
+
+var server = http.createServer(function(req, res) {
+  if (req.url === '/') {
+    res.writeHead(200, {'content-type': 'text/html'});
+    res.end(
+      '<form action="/upload" enctype="multipart/form-data" method="post">'+
+      '<input type="text" name="title"><br>'+
+      '<input type="file" name="upload" multiple="multiple"><br>'+
+      '<input type="submit" value="Upload">'+
+      '</form>'
+    );
+  } else if (req.url === '/upload') {
+    var form = new multiparty.Form();
+
+    form.on('progress', function(bytesReceived, bytesExpected) {
```

Comment:
We should probably change the case of `bytesExpected === null` in our example so the users can understand the nuances of how it can be used.

Reply:
done!